i18n/sv.yaml

```diff
@@ -0,0 +1,234 @@
+# Navigation
+
+- id: toggle_navigation
+  translation: Växla navigering
+
+- id: table_of_contents
+  translation: Innehållsförteckning
+
+- id: on_this_page
+  translation: Innehåll
+
+- id: back_to_top
+  translation: Tillbaka till toppen
+
+# General
+
+- id: related
+  translation: Relaterad
+
+- id: minute_read
+  translation: Min Läsning
+
+- id: previous
+  translation: Föregående
+
+- id: next
+  translation: Nästa
+
+- id: figure
+  translation: "Figur %d:"
+
+- id: edit_page
+  translation: Redigera den här sidan
+
+- id: powered_by
```

Comment:
Please remove  powered_by and theme_name for now - this is an unrelated change that we'd need to consider separately, and for all language packs.